src/app/components/notifier/notifier.service.js

```diff
@@ -9,19 +9,35 @@
   'use strict';
 
   angular.module('janusHangouts')
-    .service('Notifier',  Notifier);
+    .service('MuteNotifier',  MuteNotifier);
 
-  Notifier.$inject = ['$animate', 'toastr', 'ngAudio'];
+  MuteNotifier.$inject = ['$animate', 'notifications', 'ngAudio',
+    'ActionService'];
 
-  function Notifier($animate, toastr, ngAudio) {
+  function MuteNotifier($animate, notifications, ngAudio, ActionService) {
     this.info = info;
     var bell = ngAudio.load("assets/sounds/bell.ogg");
+    var noShow = {};
 
     function info(text) {
-      bell.play();
-      toastr.info(text, {
-        onShown: function() { bell.play(); },
-        timeOut: 3000
+      if (text in noShow)
```

Comment:
Mmm, now that this notifier is not generic anymore, but only related to muting messages, I would rather move the real strings inside the notifier providing functions like
```
MuteNotifier.speaking()
MuteNotifier.muted()
```